src/TTrigger.cpp

```diff
@@ -1265,8 +1267,13 @@ void TTrigger::compile()
 bool TTrigger::setScript(const QString& script)
 {
     mScript = script;
-    mNeedsToBeCompiled = true;
-    mOK_code = compileScript();
+    if (script == QString("")) {
```

Comment:
May I suggest `if (script.isEmpty()) {` instead to avoid that raw string literal?

Reply:
Adjusted